### DISCUSSION TITLE
philosophy behind quick note being a file

### DISCUSSION THREAD

**Discussion by deathblade666** [gsantner/markor#2291] [General] [answered]
Hello, I searched but didn't see anything on this if I missed something please let me know. 

With that said, just curious on what the thought is behind quick note being a dedicated file and not a directory where a new file is created and then used? 

Amazing work on the app, and thanks for any insight here!!

**Comment by gsantner** [comment, marked as answer]
One file to quickly add things to, recently added things just a few centimeters above - without switching any files.

**Reply by deathblade666** [reply]
so kinda like a single file inbox system?

**Reply by gsantner** [reply]
It's up to you how you make use of it. You may even use a different format and do whatever you like with it. 

But yes thats the main idea.

**Reply by deathblade666** [reply]
Gotcha, thanks for explaining it.

**Comment by harshad1** [comment]
> directory where a new file is created and then used?

```  
// Put this in qn.js or something
<script>
    const stamp = new Date().toISOString().replace(/\\\[-:T.\\\]/g, '').slice(0, 14);
    const path = "./" + stamp + ".md";
    window.location.href = path;
</script>  
```
When you open this in view mode it will redirect you to a timestamped file in the same directory. Create a shortcut to this file and you have what you described.

**Reply by aisbergde** [reply]
Advantage of QuickNote: it makes it very easy to collect content.

But with heavy use, QuickNote becomes very large and cluttered. Could a feature be added that allows users to highlight content and move it to a separate file? Or does this already exist? Or how do other users bring a bit more order and structure to their work with QuickNote?

**Reply by achrodo** [reply]
The feature to highlight content and move it to a separate file already exists. It's built into the OS: cut/paste.

For me, Quicknote is ephemeral. Nothing stays there very long. If any text is there for more than a day, then I must have forgot to put that text where it actually belongs. Quicknote's default state is to be empty. Like your lungs at the bottom of an exhale. They have done their job, and are ready for your next draw of oxygen.
